### DISCUSSION TITLE
Haven't gotten a Chia win yet

### DISCUSSION THREAD

**Discussion by enhanced233** [Chia-Network/chia-blockchain#3240] [Support]
Hi guys,
I have been on Chia network for a month now and i have around 20+ TB farmed yet i haven't won a single chia during this time.
I checked with my friend and even though he farms less than me he already won 2 times (he started a bit earlier than me) i can see that i have plots passed filter 1/2XX or 2/2XX and so on occasionally and its been on estimates of 9-12~ days for a while now.

so i thought i should ask here if anyone knows whats wrong with it.

thanks for the help. :)

**Comment by dorkmo** [comment]
are the drives attached to the computer directly? or are you using any sort of NAS or anything weird?

**Reply by enhanced233** [reply]
they are all connected directly, one through sata and one through usb (external)

**Reply by dorkmo** [reply]
doesnt sound like anything is obviously wrong... you might just be unlucky :( they say you might allow 4x the estimated time.

**Reply by enhanced233** [reply]
i guess so, i just wanted to make sure nothing is wrong.
thanks for the help

**Comment by nomadengineer** [comment]
Here is a discussion that might be pertinent!
https://github.com/Chia-Network/chia-blockchain/discussions/3188

**Reply by enhanced233** [reply]
I have done that check and my longest time was around 1 second from what i saw.
though i should check for warnings and errors, maybe I'm missing something.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
